packages/notification-center/src/hooks/use-notifications.hook.ts

```diff
@@ -37,9 +37,13 @@ export function useNotifications(props?: IUseNotificationsProps) {
     await mapUpdateAction(messageId, actionButtonType, status, payload, storeId);
   }
 
-  async function refetch() {
+  const refetch = useCallback(async () => {
```

Comment:
@BiswaViraj do we also need to change our notification center implementation?

Reply:
Like on the Admin panel?
I guess not, coz it's only for the scenarios where `useNotification` hook is used, and we are not using any on the web